.cirrus.yml

```diff
@@ -0,0 +1,9 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+
+task:
+  install_script: pkg install -y
+    autoconf bash binutils coreutils e2fsprogs-libuuid
+    git gmake libunwind ncurses pkgconf
```

Comment:
Scott fixed a build issue in rpmemd, so it means that libfabric is available for FreeBSD. Wouldn't it make sense to also install it here?

What about glib? It's required by benchmarks. (If it requires more changes we can enable it later)

Reply:
I will also check both of these, I just wanted to have this as a baseline.